### ISSUE TITLE
Missing French translation support in documentaton

### ISSUE THREAD

**Issue by exaucae** [CanopyTax/single-spa.js.org#500]
It is a  widely spoken language and would help lower entry barriers for many french devs. See https://single-spa.js.org/languages

**Comment by internettrans** [comment]
Hi @exaucae, can you help with the French translation? Do you know anyone else who can help with the translation? We prefer having at least two people helping with the translation.

For reference, here was the issue for adding the Russian translation: https://github.com/single-spa/single-spa.js.org/issues/369

Here's the checklist for what needs to be done. Once we confirm there are at least two people willing to help with the translation, I'll start doing them.

### Tasks for me

- [x] Decide on a language code to use. I think that `fr` is good.
- [x] Create a new repo called `fr.single-spa.js.org`. **Manually** initialize it to be the same git tree as single-spa.js.org. Do not do this by forking the repo -- a fork has a separate git tree. You can do this by initializing to empty repo (no readme, no gitignore) and then adding that as a remote to your local copy of `single-spa.js.org`:
```sh
cd single-spa.js.org
git remote add fr git@github.com:single-spa/fr.single-spa.js.org.git
git push fr master:master
```
- [x] Update docusaurus config to correct domain (example at https://github.com/single-spa/ru.single-spa.js.org/commit/b8097893831736e3920d8d22a528c4f2458cbfad)
- [x] Update CircleCI badge in readme to correct url (example at https://github.com/single-spa/ru.single-spa.js.org/commit/e6660e0cdd2afb7c340b5362240d8f2f15721f76)
- [x] Modify [`/website/static/CNAME`](https://github.com/single-spa/single-spa.js.org/blob/master/website/static/CNAME) in to be the correct domain. https://github.com/single-spa/fr.single-spa.js.org/commit/7bd77eb4e50f607dbbb32faca64db052feb0ce3e
- [x] Force HTTPS in the repository settings in Github. This takes a day or so after the CNAME is set for the SSL cert to be set up.
- [x] Create a pull request to the js.org repo, adding the subdomain. Should be similar to this: https://github.com/js-org/js.org/pull/3747
- [x] Add French to [translations.js](https://github.com/single-spa/single-spa.js.org/blob/master/website/src/data/languages.js), so that it appears in the list at https://single-spa.js.org/languages

### Tasks for @exaucae    

- [ ] Find at least one other person who can help with the translations.
- [ ] [Watch the fr.single-spa.js.org Github repository](https://docs.github.com/en/enterprise-server@2.20/github/receiving-notifications-about-activity-on-github/watching-and-unwatching-repositories) for all activity
- [ ] Translate the files `/website/versioned_docs/version-5.x` directory. Do not translate the `/docs` directory. Also, translate each of the pages in `/website/pages`

### Tasks for @guguji5

- [ ] Set up the bot to automatically commit to fr.single-spa.js.org whenever a PR is merged into single-spa.js.org.

**Comment by exaucae** [comment]
Hey @joeldenning,

I'd be glad to help, yes. Î'll let you know when I find the second person.

**Comment by exaucae** [comment]
@joeldenning , unofrtunately I have not found a second person. If that's okay for you, we can start just with me. I'll be waiting for the french repo to be set up ( or to have access to it, since https://github.com/single-spa/fr.single-spa.js.org redirect me to Github's 404 page) .  I'll also keep looking for a second person meanwhile.

**Comment by internettrans** [comment]
Sure @exaucae, I've created https://github.com/single-spa/fr.single-spa.js.org and started the process. The remaining steps require a couple days to go through, so I'll plan on coming back to this over time. Feel free to mention me if I do not finish out the remaining steps.

In the meantime while we're waiting on Github and js.org, you can boot up the french translation site locally and start translating the documentation (as described in my original post of this issue). You don't need to wait for my approval or anyone else's on PRs - just merge them.

Also, feel free to mention me here if i forget to finish out the remaining steps and don't come back to it in a few days

**Comment by internettrans** [comment]
Created https://github.com/single-spa/single-spa.js.org/pull/520 to add the link to the French documentation

**Comment by internettrans** [comment]
Created https://github.com/js-org/js.org/pull/6794 for js.org subdomain
